libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -34,11 +35,17 @@ import { merge } from 'rxjs';
 import { tap } from 'rxjs/operators';
 import { TableWrapperComponent } from '@perun-web-apps/perun/utils';
 import { TableConfigService } from '@perun-web-apps/config/table-config';
+import { MemberLifecycleAlterablePipe } from '../../../../pipes/src/lib/member-LifecycleAlterable.pipe';
```

Comment:
add this pipe to file /libs/perun/pipes/src/index.ts and the import path will be much more nicer